### ISSUE TITLE
"Could not satisfy" on `pip install`

### ISSUE THREAD

**Issue by mjbrender** [databricks/spark-pr-dashboard#52]
It seems I have trouble getting `jira-python` from following the install steps. 

```
➜  spark-pr-dashboard git:(master) ✗ pip install -r requirements.txt -t lib
Collecting Flask==0.10.1 (from -r requirements.txt (line 7))
  Using cached Flask-0.10.1.tar.gz
Collecting Flask-Cache==0.13.1 (from -r requirements.txt (line 8))
  Using cached Flask-Cache-0.13.1.tar.gz
Collecting python-dateutil==2.2 (from -r requirements.txt (line 9))
  Using cached python-dateutil-2.2.tar.gz
Collecting LinkHeader==0.4.2 (from -r requirements.txt (line 10))
  Using cached LinkHeader-0.4.2.tar.gz
Collecting jira-python==0.16 (from -r requirements.txt (line 11))
  Could not find any downloads that satisfy the requirement jira-python==0.16 (from -r requirements.txt (line 11))
  Some externally hosted files were ignored as access to them may be unreliable (use --allow-external jira-python to allow).
  No distributions at all found for jira-python==0.16 (from -r requirements.txt (line 11))
```

I'm writing a PR that uses `--allow-external` to get past this error. More to come :+1:

**Comment by mjbrender** [comment]
I no longer error when I change the jira line to `--allow-external jira-python==0.16`, but I don't see `jira-python` installed. I was successful with `pip install jira`. Is that an updated name to this requirement or something totally separate?

**Comment by nchammas** [comment]
Yes, that package was renamed and the versioning kinda messed up at one point, with backwards incompatible changes between minor releases.

I had to go through something similar. Not sure what the correct thing to do is anymore, since I went through this a while ago and don't remember.

**Comment by JoshRosen** [comment]
I'm fine with moving to the `jira` library; I think that our use of `jira-python` was a carryover from some old JIRA scraping code that was written a couple of years ago.

**Comment by nchammas** [comment]
Aren't we using it to set issue status in JIRA too? We'd have to update
that code if we upgrade our dependency (which we perhaps should do at some
point anyway).

**Comment by JoshRosen** [comment]
Yep, we'd have to test / update that code as well. Given that this is going to require some manual testing, maybe it's best if I implement this change myself and test it on one of my staging instances first.

**Comment by mjbrender** [comment]
@JoshRosen :+1:.

**Comment by JoshRosen** [comment]
I was able to fix this by depending on `jira==0.16` instead of `jira-python==0.16`.  I also had to specify an explicit dependency on `requests==2.5.0` to prevent the version from being bumped to a newer release that has some issues with AppEngine.  In the long run, we'll probably want to move to newer versions of both libraries, but the change in aa1477d should be sufficient to let us do a fresh redeploy of the currently-working code.
